app/delivery/send_to_providers.py

```diff
@@ -69,6 +69,10 @@ def send_sms_to_provider(notification):
             show_prefix=service.prefix_sms,
         )
 
+        if len(str(template)) == 0:
```

Comment:
Can we check if the string is not blank too (if it has spaces but no content) and not only if it is zero length? Something similar as proposed [in this stackoverflow response](https://stackoverflow.com/a/27982561/647641)?